frontend/src/pages/PasswordResetRequest.tsx

```diff
@@ -39,42 +39,46 @@ export default function PasswordResetRequest({ lang = "uk" }: Props) {
     setMsg(null);
 
     const v = email.trim();
+
     if (!v) return setError(t.required);
     if (!EMAIL_RE.test(v)) return setError(t.invalid);
 
-   setLoading(true);
+    setLoading(true);
     try {
-        const getCookie = (name: string) => {
-            const row = document.cookie.split("; ").find(r => r.startsWith(name + "="));
-            return row ? decodeURIComponent(row.split("=")[1]) : undefined;
-  };
-        const meta = document.querySelector('meta[name="csrf-token"]') as HTMLMetaElement | null;
-        const csrftoken = getCookie("csrftoken") ?? meta?.content;
-
-        const r = await fetch("/api/auth/password-reset/", {
-            method: "POST",
-            credentials: "include",
-            headers: {
-                "Content-Type": "application/json",
-                ...(csrftoken ? { "X-CSRFToken": csrftoken } : {}),
-            },
-            body: JSON.stringify({ email: v }),
-  });
+      const getCookie = (name: string) => {
+        const row = document.cookie.split("; ").find((r) => r.startsWith(name + "="));
+        return row ? decodeURIComponent(row.split("=")[1]) : undefined;
+      };
+
+      const meta = document.querySelector('meta[name="csrf-token"]') as HTMLMetaElement | null;
+      const csrftoken = getCookie("csrftoken") ?? meta?.content;
```

Comment:
The code retrieves a csrftoken from cookies/meta but the diff does not show it being included in the outgoing fetch headers. If required by the backend, omission will cause CSRF validation failures. Also relying only on meta or cookie without sending the header is insufficient.
Ensure CSRF token is attached to requests, e.g. add:
```
headers: {
  "Content-Type": "application/json",
  ...(csrftoken ? { "X-CSRFToken": csrftoken } : {}),
}
```